modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/components/ProductAmountVerticalText.kt

```diff
@@ -69,6 +72,8 @@ fun ProductAmountHorizontalText(
     hasBackgroundAlwaysWhite: Boolean = false,
     verticalAlignment: Alignment.Vertical = Alignment.CenterVertically,
     secondaryTextColor: Color = MaterialTheme.theme.colors.primaryText02,
+    isFocusable: Boolean = false,
+    focusRequester: FocusRequester? = null,
```

Comment:
I might be missing something but why there is `focusRequester` parameter? From what I checked it is always `null`.